packages/identifier/src/reserved.ts

```diff
@@ -1081,53 +1080,54 @@ const slurs = [
   'kafirs',
   'kike',
   'kikes',
-  'kraut',
-  'krauts',
+  'kkk',
+  'klukluxklan',
+  'muzzie',
+  'n1gga',
+  'n1gger',
+  'nazi ',
+  'negorid',
```

Comment:
This looks like a typo. Shouldn't it be `negroid`?

Reply:
Why not both?